app/src/org/commcare/activities/SyncCapableCommCareActivity.java

```diff
@@ -109,6 +112,7 @@ public void handlePullTaskResult(ResultAndError<DataPullTask.PullTaskResult> res
                 updateUiAfterDataPullOrSend(Localization.get("sync.fail.bad.data"), FAIL);
                 break;
             case DOWNLOAD_SUCCESS:
+                CommCareApplication.notificationManager().clearNotifications(NotificationMessageFactory.StockMessages.BadSslCertificate.getCategory());
```

Comment:
Thinking around if we add more error handling paths that result into error notifications from sync, it would be helpful if we can delete all notifications related to sync errros here instead of just one corresponding to `BadSslCertificate`.